### PR TITLE
pkp/pkp-lib#1498 improve crossref statuses

### PR DESCRIPTION
**Pull request by bozana**
S. pkp/pkp-lib#1498

### CODE REVIEW

File: plugins/importexport/crossref/templates/articles.tpl [hunk 1]
```diff
@@ -27,80 +27,98 @@
...
+								{if $isEditor}
+									<a href="{url page="editor" op="viewMetadata" path=$article->getId()}" class="action">{$article->getPubId('doi')|escape}</a></td>
+								{else}
+									{$article->getPubId('doi')|escape}
+								{/if}
+							<td>
+								{if $status == $smarty.const.CROSSREF_STATUS_NOT_DEPOSITED}
+									{translate key="plugins.importexport.crossref.status.non"}
+								{else}
+									<input type="hidden" name="filter" value={$filter|escape} />
```
Comment (asmecher) [inline plugins/importexport/crossref/templates/articles.tpl]: Needs quoting.

File: plugins/importexport/datacite/classes/DOIExportPlugin.inc.php [hunk 2]
```diff
@@ -161,6 +161,8 @@ function display(&$args, &$request) {
 			case 'suppFiles':
 				// Test mode.
 				$templateMgr->assign('testMode', $this->isTestMode($request)?array('testMode' => 1):array());
+				$filter = $request->getUserVar('filter');
```
Comment (asmecher) [inline plugins/importexport/datacite/classes/DOIExportPlugin.inc.php]: Is this line dead code? The line below gets from the request again.

**Comment by asmecher** [comment]
Just a couple of comments. Should this be cherry-picked into `ojs-stable-2_4_8` as well (when ready)?

**Comment by bozana** [comment]
@asmecher, this is already merged in ojs-stable-2_4_8 -- this was just a cherry-pick for dev :-) I'll make the changes according to your comments and submit a PR for them also for the ojs-stable-2_4_8 :-) THANKS!!!

**Comment by bozana** [comment]
@asmecher, I did the changes, s. the commit "fixes after code review" above. THANKS!!! :-)

**Comment by asmecher** [comment]
Looks good, thanks! Just a small PR for 2.4.8-1, correct?

**Pull request merged**